### PR TITLE
fix cryptoTransactions

### PR DESCRIPTION
**Pull request by meeyun-deriv**

### CODE REVIEW

**Review by bilalmajid-deriv — APPROVED** [review]
LGTM

File: cypress/e2e/wip/Wallets/cashier/cryptoTransactions.cy.js [hunk 1]
```diff
@@ -46,15 +46,16 @@ describe("WALL-2858 - Crypto transfer and transactions", () => {
     cy.log("View Transactions of Crypto account")
     cy.contains("Wallet", { timeout: 10000 }).should("exist")
     cy.findAllByText(/BTC Wallet/).first().scrollIntoView()
-    cy.get('.wallets-accordion__dropdown > svg > path').first().click({force: true})
+    cy.findByTestId('dt_wallets_textfield_icon_right').findByRole('button').click();
+    cy.findAllByText('BTC Wallet').first().click();
```
Comment (abuzar-deriv) [inline cypress/e2e/wip/Wallets/cashier/cryptoTransactions.cy.js:50]: Can you remove the semi-colons please 😄

File: cypress/e2e/wip/Wallets/cashier/cryptoTransactions.cy.js [hunk 2]
```diff
@@ -46,15 +46,16 @@ describe("WALL-2858 - Crypto transfer and transactions", () => {
...
-    cy.findByRole("option", { name: "Withdrawal" }).click()
-    cy.contains("No recent transactions")
-    cy.get("#downshift-0-toggle-button").findByRole("button").click()
+   cy.findByTestId('dt_wallets_textfield_box').click();
+   cy.findByRole("option", { name: "Deposit" }).click()
+   cy.contains("+5.00000000 BTC")
+   cy.findByTestId('dt_wallets_textfield_box').click();
+   cy.findByRole("option", { name: "Withdrawal" }).click()
+   cy.contains("No recent transactions")
+   cy.findByTestId('dt_wallets_textfield_box').click();
```
Comment (abuzar-deriv) [inline cypress/e2e/wip/Wallets/cashier/cryptoTransactions.cy.js:58]: Same here.

File: cypress/e2e/wip/Wallets/cashier/cryptoTransactions.cy.js (hunk 1 above)
Comment (bilalmajid-deriv) [reply cypress/e2e/wip/Wallets/cashier/cryptoTransactions.cy.js:50]: or maybe `npm install` before committing the changes

**Comment by meeyun-deriv** [comment]
closing this as I messed up with prettier

**Pull request closed without merging**